src/app/app.component.ts

```diff
@@ -1,11 +1,33 @@
-import { Component} from '@angular/core';
+import { Component, OnInit} from '@angular/core';
+import { Router, NavigationEnd, ActivatedRoute } from '@angular/router';
+import { Title } from '@angular/platform-browser';
+import { filter, map, mergeMap} from 'rxjs/operators';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
-  constructor() {
+export class AppComponent implements OnInit {
+  constructor(
+    private router: Router,
+    private activatedRoute: ActivatedRoute,
+    private titleService: Title
+  ) {}
```

Comment:
I think the general pattern is to have a newline after the opening brace, although it looks like it is not enforced in the linter.

Reply:
Ignore this; seems like we are not consistent after all.